src/xdocs/config_filters.xml

```diff
@@ -985,6 +985,27 @@ public class InputTest {
       </subsection>
     </section>
 
+    <section name="SupperssionXpathSingleFilter">
+      <subsection name="Description" id="SuppressionXpathSingleFilter">
+
+      </subsection>
+      <subsection name="Properties" id="SuppressWithXpathSingleFilter_Properties">
+
```

Comment:
All your setters in the filter are properties. You will need to add them here and document them similar to the file.
See http://checkstyle.sourceforge.net/config_filters.html#SuppressionXpathFilter_Examples for how xpath described the file.

Reply:
I wonder how should I define in the default value field? Could I made whole Regular Expression default in ".*". And String value in "Null"?